libs/features/webapp-prompt/webapp-prompt.css

```diff
@@ -148,18 +148,19 @@
   }
 
   .appPrompt-progress {
-    width: 1%;
+    width: 100%;
     height: 100%;
     animation: appPrompt-animate 5s linear forwards;
     transform-origin: 0 0;
+    transform: scaleX(0);
   }
 
   @keyframes appPrompt-animate {
     0% {
-        transform: scaleX(1);
+        transform: scaleX(0);
```

Comment:
Do we even need the `0%` in this case?

Reply:
Good point. The animation will start from the initial state anyway. Made the change 👍